CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.13)
+project(WickedEngine)
+
+set(CMAKE_CXX_STANDARD 17)
+set(VULKAN_SDK_DIR "/home/jeremy/VulkanSDK/1.2.135.0/x86_64")
```

Comment:
That's probably not a good path. On Windows I set it from a system variable which is created by Vulkan SDK installer: $(VULKAN_SDK)/Include 
Is there something like that for Linux?

Reply:
Done. On Linux, you now have to export VULKAN_SDK=".../path/to/vulkan_sdk"